src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -599,7 +599,7 @@ public SscsCaseData(@JsonProperty(value = "ccdCaseId", access = JsonProperty.Acc
         this.adjournCaseNextHearingSpecificDate = adjournCaseNextHearingSpecificDate;
         this.adjournCaseNextHearingSpecificTime = adjournCaseNextHearingSpecificTime;
         this.adjournCaseReasons = adjournCaseReasons;
-        this.adjournCaseAnythingElse = adjournCaseAnythingElse;
+        this.adjournCaseAdditionalDirections = adjournCaseAnythingElse;
```

Comment:
could you rename the field to the right?

Reply:
done - thanks Jack